src/data/faq_de.json

```diff
@@ -93,8 +93,8 @@
                             "Das Sicherheitszertifikat <b>T-Systems Enterprise Services GmbH, T-TeleSec GlobalRoot Class 2</b> ist möglicherweise auf Ihrem Gerät deaktiviert. Dies ist eine bekannte Ursache, um folgende Benachrichtigung auszulösen:",
                             "<b>'URSACHE: 2001 Etwas ist schiefgelaufen. An error occurred while trying to establish a secure connection to the server.'</b>",
                             "Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",

```

Comment:
Suggest possible change from:

"Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren (Der genaue Pfad hängt vom Gerät ab).",

to:

"Führen Sie die folgenden Schritte aus, um das Sicherheitszertifikat zu aktivieren:",
"(Der genaue Pfad hängt vom Gerät ab).",

Reply:
see above